awx/ui/client/src/login/loginModal/loginModal.partial.html

```diff
@@ -98,7 +98,11 @@
                         </div>
                     </div>
                 </form>
-                <div id="login_modal_notice" class="LoginModalNotice" ng-if="customLoginInfoPresent"><div class="LoginModalNotice-title" translate>NOTICE</div>{{ customLoginInfo | sanitize }}</div>
+                <div id="login_modal_notice" class="LoginModalNotice" ng-if="customLoginInfoPresent"
+                    ng-style="{'white-space' : (customLoginInfoIsHTML) ? 'normal': 'pre-wrap'}">
```

Comment:
The reason I added the `customLoginInfoIsHTML` check to switch between `normal` and `pre-wrap` is because the original state of the element before this PR was `pre-wrap`, as you can see in [/awx/ui/client/src/login/loginModal/loginModalNotice.block.less#L15](https://github.com/ansible/awx/blob/d0ac02826537343880b09282666639cf5135779c/awx/ui/client/src/login/loginModal/loginModalNotice.block.less#L15).

(Technically, now that this PR sets the value, that line could be eliminated!)

I think I forgot to force-reload after my change to verify the plain text -- I'm seeing that it behaves oddly now as well. In your case, since you only had a single line of plain text, it didn't matter that you used `normal` white-space, but if you had multi-line plain text, it would wrap it into one line.

Trying to switch to `normal` was *my* design decision -- there's an argument to be made for leaving the CSS alone so that people could use simple `<em>` and `<b>` markup, and still have lines and wrapping done properly without having to use `<br />` or `<p>` tags. The only downside is that those of us who want to dump larger HTML fragments like this:
```html
<p>
Please review our <a href="https://global.intranet.company.com/homesites/GDPRHOME/GDPR%20documents/Company%20Internal%20Technologies%20Privacy%20Notice_version%2019062019.pdf">GDPR Privacy Statement</a>.
</p>
<p>
For your reference:
<ul>
<li><a href="https://global.intranet.company.com/confluence/x/oRQQw">User Guide</a</li>
<li><a href="https://global.intranet.company.com/confluence/display/AWX/FAQ">FAQ</a</li></ul>
</p>
```
would have to reformat them to be line-oriented like this:
```html
Please review our <a href="https://global.intranet.company.com/homesites/GDPRHOME/GDPR%20documents/Company%20Internal%20Technologies%20Privacy%20Notice_version%2019062019.pdf">GDPR Privacy Statement</a>.

For your reference:<ul><li><a href="https://global.intranet.company.com/confluence/x/oRQQw">User Guide</a</li><li><a href="https://global.intranet.company.com/confluence/display/AWX/FAQ">FAQ</a</li></ul>

Reply:
let me know what you think @jakemcdermott, and I will look deeper into how to properly manage the css for this element.

Reply:
Ok, I figured it out -- originally the custom info was templated directly into the `div#login_modal_notice` container, so the CSS needed to be applied to that element. Since I'm using ng-bind-html, a new element is created in that div, so that's where the CSS needs to apply. So I removed the style from `loginModalNotice.block.less` and set it on the `<ng-bind-html>` element dynamically instead.

There's now three commits, but I can squash them as you wish.

I've also rebased on latest devel.